explorer/templates/explorer/query.html

```diff
@@ -8,7 +8,13 @@
             <h2>
                 {% if query %}
                     {{ query.title }}
+                    {% if is_favorite %}
```

Comment:
This icon is a bit hidden until you know it's there - if that make sense.

Maybe it should be red to make it stand out?

I also think it might be good if the query list on the main page had a column for marking queries as favourites.

Reply:
Ok made icon red, see what you think. Could we handle marking favorite on main page in separate enhancement? It'll probably be next week before I could get to that since it's a little more complicated and I want to get this initial feature tied up this weekend before I get back busy with work if ok with you?

Reply:
@lawson89 thanks. I got half way through a response & then ended up busy. No rush here - it's looking good. Thank you!